extensions/amp-youtube/0.1/amp-youtube.js

```diff
@@ -181,14 +190,11 @@ class AmpYoutube extends AMP.BaseElement {
     this.element.appendChild(iframe);
 
     this.iframe_ = iframe;
-    this.playerReadyPromise_ = new Promise(resolve => {
-      this.playerReadyResolver_ = resolve;
-    });
 
-    const ampdoc = this.getAmpDoc();
-    ampdoc.win.addEventListener(
+
+
+    this.win.addEventListener(
```

Comment:
I think this is accurate but please test the `/pwa/` and `/a4a/` cases and make sure it is behaving as expected.